sdk/translation/Azure.AI.Translation.Text/src/TextTranslationClientBuilderExtensions.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using Azure;
+using Azure.AI.Translation.Text;
+using Azure.Core;
+using Azure.Core.Extensions;
+
+//TODO: there is no way to only suppress a single memmber of a static class so we need to have everything custom here.
```

Comment:
Which member do you want to suppress?

Reply:
There were DI registration of some versions of Client that we do not support. As it was static method, there was no way to override it in different way.